src/main/webapp/i18n/de/error.json

```diff
@@ -20,6 +20,8 @@
         "quizNotEndedYet": "Das Quiz hat noch nicht geendet.",
         "quizAlreadyOpenForPractice": "Das Quiz ist bereits zur Übung geöffnet.",
         "unknownAction": "Diese Aktion kann nicht durchgeführt werden.",
-        "groupNotFound": "Die von dir eingegebene Gruppe wurde nicht gefunden."
+        "groupNotFound": "Die von dir eingegebene Gruppe wurde nicht gefunden.",
+        "cantSaveFile": "Die hochgeladene Datei {{ fileName }} konnte nicht gespeichert werden.",
+        "emptyFile": "Die hochgeladene Datei {{ fileName }} ist leer. Bitte laden Sie eine andere Datei hoch"
```

Comment:
We should be consistent with the salutation. Either use "du" or "Sie" in German. I think we are already using "du" most of the times, so I would change it here.

Reply:
You are right here. I use `du` now